isnad.csl

```diff
@@ -0,0 +1,1387 @@
+      <term name="and others">v.dğr.</term>
+      <term name="and others" form="short">v.dğr.</term>
+      <term name="issue" form="short">sy.</term>
+      <term name="translator" form="short">trc.</term>
+      <term name="translator" form="verb-short">trc.</term>
+      <term name="and"> - </term>
+      <term name="edition"/>
+      <term name="collection-editor" form="short">thk.</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="verb-short">ed.</term>
+      <term name="translator" form="verb-short">trans.</term>
+      <term name="translator" form="short">trans.</term>
+      <term name="editortranslator" form="verb-short">
+        <single>ed. and trans.</single>
+        <multiple>ed. and trans.</multiple>
+      </term>
+      <term name="editortranslator" form="verb">
+        <single>Edited and translated by</single>
+        <multiple>Edited and translated by</multiple>
+      </term>
+      <term name="translator" form="short">trans.</term>
+      <term name="and"> - </term>
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <group delimiter=" ">
+        <choose>
+          <if variable="container-author reviewed-author collection-editor editorial-director" match="any">
+            <group>
+              <group prefix="thk. ">
+                <names variable="collection-editor">
+                  <name delimiter=" - " and="text"/>
+                </names>
+              </group>
+              <group prefix="nşr. ">
+                <names variable="editorial-director">
+                  <name delimiter=" - " and="text"/>
+                </names>
+              </group>
+              <names variable="container-author reviewed-author">
+                <label form="verb-short" text-case="lowercase" suffix=" "/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </group>
+          </if>
+        </choose>
+      </group>
+      <names variable="editor translator" delimiter=", ">
+        <label form="verb-short" text-case="lowercase" suffix=" "/>
+        <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+      </names>
+    </group>
+  </macro>
+  <macro name="secondary-contributors-note">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-contributors-note">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text macro="editor-translator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <group prefix="Thk. ">
+          <names variable="collection-editor">
+            <name delimiter=" - " and="text"/>
+          </names>
+        </group>
+        <group prefix="Nşr. ">
```

Comment:
I'm not sure what exactly these prefixes are, but if the style requires that you include hardcoded Turkish, it can't be a multilingual style (i.e. you'd have to set a default-locale and could remove the English locale definitions)